tests/test_seaexplorer.py

```diff
@@ -91,35 +91,35 @@ def test_raw_to_timeseries():
     with pytest.raises(FileNotFoundError) as missing_file_exc:
         result_default = seaexplorer.raw_to_timeseries('tests/data/realtime_rawnc/',
                                                         'tests/data/l0-profiles/',
-                                                        example_dir / 'example-seaexplorer/deploymentRealtime.yml',
+                                                        str(example_dir / 'example-seaexplorer/deploymentRealtime.yml'),
                                                         )
     result_sub = seaexplorer.raw_to_timeseries('tests/data/realtime_rawnc/',
                                                     'tests/data/l0-profiles/',
-                                                    example_dir / 'example-seaexplorer/deploymentRealtime.yml',
+                                                    str(example_dir / 'example-seaexplorer/deploymentRealtime.yml'),
                                                     kind='sub')
     assert 'No such file or directory' in str(missing_file_exc)
     assert result_sub == 'tests/data/l0-profiles/dfo-eva035-20190718.nc'
 
 
 def test_missing_bad_timebase():
     # Prepare yaml files with bad timebase and no timebase
-    with open(example_dir / 'example-seaexplorer/deploymentRealtime.yml') as fin:
+    with open(str(example_dir / 'example-seaexplorer/deploymentRealtime.yml')) as fin:
```

Comment:
Can you explain why this is needed?  `open` should work with Path variables.

Reply:
> @cassandra-elmer do you mean getting the files to pass the IOOS compliance checker?



The tests I meant are these

![Screenshot 2024-07-12 132136](https://github.com/user-attachments/assets/2c64a03c-dda3-45b7-9d0b-5a7b2fb7a95f)

Working on the compliance checker is my own next step, but I wanted to fix the above tests first.



> Can you explain why this is needed? `open` should work with Path variables.



Those are likely redundant because I got trigger happy, yes. There were previous `example_dir / '*.yml` spots that weren't `open` and were breaking and I just carried it through.